src/main/java/gui/TimetableBlock.java

```diff
@@ -5,68 +5,139 @@
 import javafx.geometry.Insets;
 import javafx.geometry.Point2D;
 import javafx.scene.Cursor;
+import javafx.scene.Node;
 import javafx.scene.Parent;
+import javafx.scene.SnapshotParameters;
 import javafx.scene.control.Label;
+import javafx.scene.effect.BlendMode;
+import javafx.scene.effect.ColorAdjust;
+import javafx.scene.effect.DropShadow;
+import javafx.scene.effect.GaussianBlur;
+import javafx.scene.image.ImageView;
+import javafx.scene.image.WritableImage;
 import javafx.scene.input.MouseEvent;
 import javafx.scene.layout.AnchorPane;
 import javafx.scene.layout.GridPane;
 import javafx.scene.layout.Pane;
 import javafx.scene.layout.Region;
 import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
 import javafx.scene.shape.Rectangle;
 import lombok.Getter;
 
 /**
- * Class that resembles a draggable, resiable block inside the timetable.
- * Highly volatile. Do not poke too much.
+ * Class that resembles a draggable, resiable block inside the timetable,
+ * whose sole purpose is to display information.
+ * Highly volatile. Do not poke the dragging-dragon too much.
  */
 public class TimetableBlock extends Pane {
 
     public enum DraggingTypes { Move, Resize_Top, Resize_Right, Resize_Bottom, Resize_Left }
 
     /*
         Styling variables.
-        For styling and displayable content.
+        For tweaking the styling.
      */
 
-    private String normalStyle = "-fx-border-style: solid inside;"
-            + "-fx-border-width: 3;"
-            + "-fx-border-color: yellow;"
-            + "-fx-background-color: orange;";
-    private String dragStyle = "-fx-border-style: solid inside;"
-            + "-fx-border-width: 3;"
-            + "-fx-border-color: red;"
-            + "-fx-background-color: orange;";
+    private double verticalBorderSize = 4.0;
+    private double margin = 5.0;
+    private double blurRadius = 20.0;
 
-    private String title = "dummyTitle with some extension";
+    private String colorBlockBackground = "#B1E2FA";
+    private String colorBlockBorders = "#DDF2FC";
+    private String colorBlockBorderHighlight = "#005482";
+    private String colorText = "#047FB7";
+
+    /*
+        Temporary variables, until they can be moved to CSS files properly.
+     */
+
+    private String normalStyleBackground =
```

Comment:
Is it possible to extract these style rules to different classes or maybe even files to prevent cluttering this class?